### ISSUE TITLE
example that does not converge

### ISSUE THREAD

**Issue by targos** [mljs/fcnnls#3]
With matlab and python implementations, the following example returns `[0, 50, 10]`.
With `ml-fcnnls`, it never converges (or exceeds the maximum number of iterations).

```js
const { fcnnlsVector } = require("ml-fcnnls");

const data = JSON.parse(
  '{"mC":[[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0.0001,0,0.0001],[0.0001,0,0.0001],[0.0002,0,0.0002],[0.0005,0,0.0005],[0.0009,0,0.0009],[0.0016,0,0.0016],[0.0027,0,0.0027],[0.0045,0,0.0045],[0.0071,0,0.0071],[0.0108,0,0.0108],[0.0158,0,0.0158],[0.0222,0,0.0222],[0.0299,0,0.0299],[0.0388,0,0.0388],[0.0484,0,0.0484],[0.0579,0,0.0579],[0.0666,0,0.0666],[0.0737,0,0.0737],[0.0782,0,0.0782],[0.0798,0.0002,0.0802],[0.0782,0.0006,0.0794],[0.0737,0.0021,0.0779],[0.0666,0.0059,0.0784],[0.0579,0.0152,0.0883],[0.0484,0.035,0.1184],[0.0388,0.072,0.1828],[0.0299,0.1326,0.2951],[0.0222,0.2187,0.4596],[0.0158,0.3226,0.661],[0.0108,0.4259,0.8626],[0.0071,0.5032,1.0135],[0.0045,0.5319,1.0683],[0.0027,0.5032,1.0091],[0.0016,0.4259,0.8534],[0.0009,0.3226,0.6461],[0.0005,0.2187,0.4379],[0.0002,0.1326,0.2654],[0.0001,0.072,0.1441],[0.0001,0.035,0.0701],[0,0.0152,0.0304],[0,0.0059,0.0118],[0,0.0021,0.0042],[0,0.0006,0.0012],[0,0.0002,0.0004],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0],[0,0,0]],"bf":[0,0,0,0,0,0.001,0.001,0.002,0.005,0.009,0.016,0.027000000000000003,0.045,0.07100000000000001,0.10800000000000001,0.15800000000000003,0.222,0.299,0.388,0.484,0.579,0.666,0.737,0.782,0.8119999999999999,0.8240000000000001,0.8839999999999999,1.079,1.643,2.934,5.427999999999999,9.581,15.531,22.74,29.921000000000003,35.295,37.278000000000006,35.251000000000005,29.829,22.591,15.314,9.284,5.0409999999999995,2.451,1.064,0.413,0.147,0.041999999999999996,0.014,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]}'
);

const result = fcnnlsVector(data.mC, data.bf);

console.log(result);
```

**Comment by thegodone** [comment]
Did it works now ?

**Comment by targos** [comment]
Fixed in 1.1.4
